### DISCUSSION TITLE
net returned to the grid

### DISCUSSION THREAD

**Discussion by lukibazuki** [flixlix/energy-gauge-bundle-card#6] [Q&A] [answered]
Hi there

Could it be possible that due to this add the "net returned to the grid" (type: energy-grid-neutrality-gauge) card goes missing from the Energy Dashboard? I've just done the HA update today, so it could be a coincidence...

Thank You

**Comment by flixlix** [comment, marked as answer]
Yes, that is fortunately a known bug, but it will be fixed in the next version :)
